src/item.rs

```diff
@@ -347,6 +347,32 @@ impl Item {
             _ => unreachable!(),
         }
     }
+
+    #[cfg(feature = "parsing")]
+    pub fn attrs(&self) -> Option<&Vec<Attribute>> {
```

Comment:
I have a similar implementation of this in a crate here: [parse_attributes.rs#L336](https://github.com/jf2048/deluxe/blob/cdc455fe3090aaa7215fe7dd676c07814e195045/core/parse_attributes.rs#L336)

I was able to have it simply return `&[Attribute]`, and for `Item::Verbatim` it just returns `&[]`. Not sure if that is better or not.